pkg/vpcmodel/connectivityAbstraction.go

```diff
@@ -152,8 +120,66 @@ func mergeConnectivityWithNodeSetAbstraction(
 		for _, conns := range nodeConns {
 			allConns = allConns.Union(conns)
 		}
-		res.updateAllowedConnsMap(src, nodeSet, allConns)
+		mergedConnectivity.updateAllowedConnsMap(src, nodeSet, allConns)
 	}
+	return mergedConnectivity
+}
 
-	return res
+// nodeSetAbstractionInformation() collects abstraction information of the nodeSet.
+// for now, it collects the connections that are missing for full abstraction.
+func (nsa *NodeSetAbstraction) nodeSetAbstractionInformation(mergedConnectivity GeneralConnectivityMap,
+	nodeSetToNodeSet, otherFromNodeSet, otherToNodeSet GeneralConnectivityMap,
+	nodeSet NodeSet) *AbstractionInfo {
+	abstractionInfo := &AbstractionInfo{}
+	abstractionInfo.missingEgressConnections = nsa.missingConnections(otherFromNodeSet, mergedConnectivity, nodeSet, false)
+	abstractionInfo.missingIngressConnections = nsa.missingConnections(otherToNodeSet, mergedConnectivity, nodeSet, true)
+	abstractionInfo.missingIngressConnections.addMap(nsa.missingConnections(nodeSetToNodeSet, mergedConnectivity, nodeSet, true))
+	return abstractionInfo
+}
+
+// missingConnections() is called on each of the last three groups.
+// it looks for the connections that are not exist in the group, but reflated in the mergedConnMap
```

Comment:
```suggestion
// it looks for the connections that do not exist in the group, but are reflated in the mergedConnMap
```

Reply:
Please add in the documentation a "practical" example of a missing connection and what it actually stands for

Reply:
done